superset/db_engine_specs/postgres.py

```diff
@@ -21,6 +21,7 @@
 from typing import Any, Dict, List, Optional, Pattern, Tuple, TYPE_CHECKING
 
 from flask_babel import gettext as __
+from psycopg2.extensions import binary_types, string_types
```

Comment:
I think this import should be moved into `get_datatype`, as `psycopg2` isn't a required dependency